File: src/main/java/org/dxworks/gitsecond/ChangeData.java
package org.dxworks.gitsecond;

import lombok.*;
import org.eclipse.jgit.diff.DiffEntry;

@Data
@Builder
@EqualsAndHashCode
@NoArgsConstructor
@AllArgsConstructor(access = AccessLevel.PACKAGE)
public class ChangeData {

    private String commitID;
    private String oldFileName;
    private String newFileName;
    private DiffEntry.ChangeType type;
    private String diff;

    private int addedLines;
    private int deletedLines;
}

File: src/main/java/org/dxworks/gitsecond/data/ChangeData.kt
package org.dxworks.gitsecond.data

import org.eclipse.jgit.diff.DiffEntry


data class ChangeData(var commitID: String, var oldFileName: String, var newFileName: String, var type: DiffEntry.ChangeType, var diff: String = "", var annotatedLines: MutableList<AnnotatedLineData>) {

}
